pom.xml

```diff
@@ -535,9 +535,20 @@
             <configuration>
               <excludes>
                 <exclude>**/CudaFatalTest.java</exclude>
+                <exclude>**/ColumnViewNonEmptyNullsTest.java</exclude>
               </excludes>
             </configuration>
           </execution>
+          <execution>
+            <id>non-empty-null-test</id>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <argLine>-da:ai.rapids.cudf.AssertEmptyNulls</argLine>
+              <test>ColumnViewNonEmptyNullsTest</test>
+            </configuration>
+          </execution>
```

Comment:
This new execution is added however all tests are excluded:
```
3:56:31  [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.0:test (non-empty-null-test) on project spark-rapids-jni: No tests were executed!  (Set -DfailIfNoTests=false to ignore this error.) -> [Help 1]
```

Reply:
How come? The documentation says it will add each test https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#test

Reply:
CLI properties override pom

Reply:
hmm... but that would not even run CudaFatalTest but we see in previous successful premerge that it does run the [CudaFatalTest](https://blossom.nvidia.com/sw-gpu-spark-jenkins/job/rapids-jni_premerge-github/243/console)

Reply:
I think it cannot find the Test because it's pointing to an older cudf which doesn't have that Test. I wonder how we expect this to work if we aren't supposed to commit the submodule change

Reply:
OK. So I got confused when @jlowe said we aren't supposed to push changes to the submodule. 
Thanks @pxLi for clarifying